.github/workflows/bundle-update.yml

```diff
@@ -0,0 +1,59 @@
+name: Scheduled Bundle Update PR
+
+on:
+  schedule:
+    - cron: '0 0 * * 2'
```

Comment:
[Dependabot supports weekly scheduling and even arbitrary `cron` too](https://docs.github.com/en/code-security/dependabot/working-with-dependabot/dependabot-options-reference#schedule-), if you’re looking to update the Lockfile less frequently.